mne/viz/tests/test_3d.py

```diff
@@ -391,6 +393,10 @@ def test_process_clim_plot(renderer_interactive):
     stc.plot(colormap='hot', clim='auto', **kwargs)
     stc.plot(colormap='mne', clim='auto', **kwargs)
     stc.plot(clim=dict(kind='value', lims=(10, 50, 90)), figure=99, **kwargs)
+    # figure has been set so the internal ref needs to be clean to avoid
+    # lingering VTK objects in memory
+    if is_pyvista:
+        renderer_interactive.backend._FIGURES.clear()
```

Comment:
I don't fully understand -- doesn't this indicate that the `close_all` that happens at the end of this test is somehow not working? How do we get a figure set/created in a way that our close-all fixture will not close it?

Reply:
We keep an internal reference to mimic `mayavi`'s behaviour when `figure` (int) is used.

Reply:
See follow-up comment -- I think this means `close_all` needs to be updated, or that we need to update our fixture to call another `close_all` for the case of int-passing or something.